committee-steering/governance/sig-governance.md

```diff
@@ -201,64 +201,27 @@ Other roles...
 
 ### Subproject Creation
 
-#### Option 1: by SIG Technical Leads
-
-Subprojects may be created by [KEP] proposal and accepted by [lazy-consensus]
-with fallback on majority vote of SIG Technical Leads.  The result *SHOULD* be
-supported by the majority of SIG Leads.
-- KEP *MUST* establish subproject owners.
+Subprojects may be created with a simple majority vote of SIG Technical Leads.
```

Comment:
For follow-up: I'm not sure if this is consistently how SIGs handle this today, we may need to broadcast this to chairs&Tls and github admins.